paprika/setup.py

```diff
@@ -31,732 +26,685 @@ def _get_installed_benchmarks():
-        destination_pdb = self.directory.joinpath(f"{self.host}-{self.guest}.pdb")
-
-        if not self.guest == "release":
-            # Align the host-guest complex so the first guest atom is at (0, 0, 0) and the second guest atom lies
-            # along the positive z-axis.
-            guest_angle_restraint_mask = self.guest_yaml["restraints"]["guest"][-1]["restraint"][
-                "atoms"
-            ].split()
-            aligned_structure = align.zalign(
-                structure, guest_angle_restraint_mask[1], guest_angle_restraint_mask[2]
-            )
-            aligned_structure.save(str(intermediate_pdb), overwrite=True)
+        if n_pull_windows is not None:
+            assert n_attach_windows is not None
 
-        else:
-            # Create a PDB file just for the host.
-
-            host = pmd.load_file(str(input_pdb), structure=True)
-            host_coordinates = host[f":{self.host_yaml['resname'].upper()}"].coordinates
-            # Cheap way to get the center of geometry
-            offset_coordinates = pmd.geometry.center_of_mass(host_coordinates,
-                                                             masses=np.ones(len(host_coordinates)))
-
-            # Find the principal components, take the two largest, and find the vector orthogonal to that
-            # (should be cross-product right hand rule, I think). Use that vector to align with the z-axis.
-            # This may not generalize to non-radially-symmetric host molecules.
-
-            aligned_coords = np.empty_like(structure.coordinates)
-            for atom in range(len(structure.atoms)):
-                aligned_coords[atom] = structure.coordinates[atom] - offset_coordinates
-            structure.coordinates = aligned_coords
-
-            inertia_tensor = np.dot(structure.coordinates.transpose(), structure.coordinates)
-            eigenvalues, eigenvectors = np.linalg.eig(inertia_tensor)
-            order = np.argsort(eigenvalues)
-            axis_3, axis_2, axis_1 = eigenvectors[:, order].transpose()
-
-            dummy_axis = np.cross(axis_1, axis_2)
-
-            self._add_dummy_to_PDB(input_pdb=input_pdb,
-                                   output_pdb=intermediate_pdb,
-                                   offset_coordinates=offset_coordinates,
-                                   dummy_atom_tuples=[(0, 0, 0),
-                                                      (dummy_axis[0], dummy_axis[1], dummy_axis[2])])
-            structure = pmd.load_file(str(intermediate_pdb), structure=True)
-
-            for atom in structure.atoms:
-                atom.mass = 1.0
-
-            aligned_structure = align.zalign(
-                structure, ":DM1", ":DM2"
+        # restraint_schemas = self.host_yaml["restraints"]["static"]
+
+        static_restraints: List[DAT_restraint] = []
+
+        n_windows = [n_attach_windows, n_pull_windows, n_release_windows]
+
+        for restraint_schema in restraint_schemas:
```

Comment:
When I get to this point, I'm not clear how we know we are grabbing the correct `restraint_schema`...

Reply:
Basically these are provided by `evaluator`, which in turn just grabs the values passed here from taproom from the correct section. E.g. here the `restraint_schema` are just the 'restraints -> static' section taken from the `taproom` host YAML files

Reply:
Got it, thanks—this helped.